nltk/sem/relextract.py

```diff
@@ -30,10 +30,10 @@
 # Dictionary that associates corpora with NE classes
 NE_CLASSES = {
     'ieer': ['LOCATION', 'ORGANIZATION', 'PERSON', 'DURATION',
```

Comment:
Because of containment checks, it would be nice to use a set instead of a list.

I guess the order of classes is not important here.